### ISSUE TITLE
[bug] no method named `get_window`

### ISSUE THREAD

**Issue by elibroftw** [tauri-apps/tauri#3784]
### Describe the bug

I was trying to make a function that shows the window in the setup of the tauri. I was trying to follow https://tauri.studio/docs/guides/splashscreen/#waiting-for-rust, but got:

error[E0599]: no method named `get_window` found for mutable reference `&mut tauri::App` in the current scope
  --> src\main.rs:30:31
   |
30 |         let main_window = app.get_window("main")?;
   |                               ^^^^^^^^^^ method not found in `&mut tauri::App`

### Reproduction

```rs
fn main() {
  tauri::Builder::default()
    // .invoke_handler(tauri::generate_handler![custom_command])
    .setup(|app| {
        let main_window = app.get_window("main")?;
        main_window.show()?
    })
    // https://github.com/tauri-apps/tauri-plugin-window-state/issues/21
    // .plugin(tauri_plugin_window_state::WindowState::default())  // requires visible: false
    .run(tauri::generate_context!())
    .expect("error while running tauri application");
}
```

### Expected behavior

Compiles

### Platform and versions

```shell
PS C:\Users\maste\Documents\GitHub\r2-t2> yarn tauri info
yarn run v1.22.11
$ C:\Users\maste\Documents\GitHub\r2-t2\node_modules\.bin\tauri info

Operating System - Windows, version 10.0.19044 X64Webview2 - 99.0.1150.52
Visual Studio Build Tools:
   - Visual Studio Community 2022

Node.js environment
  Node.js - 17.3.1 
  @tauri-apps/cli - 1.0.0-rc.6
  @tauri-apps/api - 1.0.0-rc.2

Global packages
  npm - 8.3.0
  pnpm - Not installed
  yarn - 1.22.11

Rust environment
  rustup - 1.24.3
  rustc - 1.57.0
  cargo - 1.57.0
  toolchain - stable-x86_64-pc-windows-msvc 

App directory structure
/.git
/assets
/build
/node_modules
/public
/pyi-specs
/src
/src-tauri
/__pycache__

App
  tauri - 1.0.0-rc.4
  tauri-build - 1.0.0-rc.4
  tao - 0.6.4
  wry - 0.13.3
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://127.0.0.1:3000/
  framework - React
Done in 10.37s.
```
```


### Stack trace

```shell
Compiling app v0.1.0 (C:\Users\maste\Documents\GitHub\r2-t2\src-tauri)
warning: unused imports: `collections::BTreeMap`, 
`fs`
 --> src\main.rs:7:11
  |
7 | use std::{collections::BTreeMap, fs};
  |           ^^^^^^^^^^^^^^^^^^^^^  ^^
  |
  = note: `#[warn(unused_imports)]` on by default 

warning: unused import: `toml`
 --> src\main.rs:8:5
  |
8 | use toml; // 0.5.1
  |     ^^^^

warning: unused import: `tauri_plugin_window_state`
  --> src\main.rs:10:5
   |
10 | use tauri_plugin_window_state;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no method named `get_window` found for mutable reference `&mut tauri::App` in the current scope
  --> src\main.rs:29:31
   |
29 |         let main_window = app.get_window("main")?;
   |                               ^^^^^^^^^^ method not found in `&mut tauri::App`
   |
   = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
6  | use tauri::Manager;
   |

For more information about this error, try `rustc 
--explain E0599`.
warning: `app` (bin "app") generated 3 warnings   
error: could not compile `app` due to previous error; 3 warnings emitted
```
```


### Additional context

_No response_

**Comment by FabianLars** [comment]
The error message actually includes the solution here:
```
error[E0599]: no method named `get_window` found for mutable reference `&mut tauri::App` in the current scope
  --> src\main.rs:29:31
   |
29 |         let main_window = app.get_window("main")?;
   |                               ^^^^^^^^^^ method not found in `&mut tauri::App`
   |
   = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
6  | use tauri::Manager;
   |
```

So, just add the missing import (`use tauri::Manager;`) at the top of the file.

**Comment by yutotakano** [comment]
By the way, I experienced this error after migrating to Tauri v2 Beta, where `use tauri::Manager` did not help. 

Turns out, this method has been renamed, see [this comment](https://github.com/tauri-apps/tauri/issues/9400#issuecomment-2052173983) and [this migration doc](https://beta.tauri.app/start/upgrade--migrate/from-tauri-1/#multiwebview-support).

Somehow missed it, a lot of the existing tutorials and plugin READMEs use v1 so I got confused.
